lib/core/core.js

```diff
@@ -76,6 +77,7 @@ axe.reset = reset;
 axe._runRules = runRules;
 axe.runVirtualRule = runVirtualRule;
 axe.run = run;
+axe.standards = standards;
```

Comment:
This will conflict with the Attest browser script, as it exposes its rulesets as `attest.standards`. Can we use a different name here?

Reply:
Also, why make this part of the public API? Now that we have a module system, we can just `import` it when necessary.

Reply:
For context, the Attest script does something like this (contrived/pseudocode):

```
import axe
import standards

var attest = {
  ...axe,
  standards,
  foo: 'bar'
}

export default attest
```

Reply:
Agreed, there's no reason to put this on the global object.